candidate/views_admin.py

```diff
@@ -903,19 +905,361 @@ def candidate_retrieve_photos_view(request, candidate_id):
     return HttpResponseRedirect(reverse('candidate:candidate_edit', args=(candidate_id,)))
 
 
+@login_required
+def candidate_merge_process_view(request):
+    """
+    Process the merging of two candidates
+    :param request:
+    :return:
+    """
+    authority_required = {'verified_volunteer'}  # admin, verified_volunteer
+    if not voter_has_authority(request, authority_required):
+        return redirect_to_sign_in_page(request, authority_required)
+
+    candidate_campaign_manager = CandidateCampaignManager()
+
+    candidate1_we_vote_id = request.POST.get('candidate1_we_vote_id', 0)
+    candidate2_we_vote_id = request.POST.get('candidate2_we_vote_id', 0)
+    google_civic_election_id = request.POST.get('google_civic_election_id', 0)
+    redirect_to_candidate_list = request.POST.get('redirect_to_candidate_list', False)
+    remove_duplicate_process = request.POST.get('remove_duplicate_process', False)
+    state_code = request.POST.get('state_code', '')
+
+    candidate1_on_stage = {}
+    candidate2_on_stage = {}
+
+    candidate1_results = candidate_campaign_manager.retrieve_candidate_campaign_from_we_vote_id(candidate1_we_vote_id)
+    if candidate1_results['candidate_campaign_found']:
+        candidate1_on_stage = candidate1_results['candidate_campaign']
```

Comment:
I would create a variable "candidate1_id = candidate1_on_stage.id" here for use below. (and do the same for candidate2_id)